services/ui-src/src/types/formFields.ts

```diff
@@ -93,6 +93,11 @@ export interface FormField {
 export function isFieldElement(
   field: FormField | FormLayoutElement
 ): field is FormField {
+  /*
```

Comment:
Ah yeah Ben and I discussed this a few days ago. Gonna leave it for right now in this PR but let's write it up as an item of tech debt. Really instead of extending fields to be either FormField or LayoutElements we should just have the form accept elements, be they fields or layout elements. But it would be a larger lift than is appropriate for this PR.
@benmartin-coforma @cassandradanger